python/external_config/external_command.py

```diff
@@ -59,13 +61,44 @@ def serialize_command(cls, entity_type, command_name, properties):
             "group_default": properties.get("group_default") or False,
 
             # special for shotgun
-            "deny_permissions": properties.get("deny_permissions"),
-            "deny_platforms": properties.get("deny_platforms"),
```

Comment:
Just checking: `deny_platforms` is not serialized anymore and now fully dynamic, right?

Reply:
good point! the caches are assumed to be per-os, so we are 'baking' the state of the current os into the cache. adding a comment to clarify!